setup.py

```diff
@@ -58,10 +58,10 @@
 
         # tracing deps
         'opentracing>=1.1,<2',
-        'opentracing_instrumentation>=2,<3',
+        'opentracing_instrumentation>=2.3,<3'
```

Comment:
we've removed the comma here.  The setup.py can't parse the dependencies here.  We should add back the commas to the end of each dependency.  This is causing the tests to fail (apart from the pypy tests which appear to be always failing).